docs/0.3/using_purchase_order.md

```diff
@@ -0,0 +1,327 @@
+    ```
+
+### Set Environment Variables
+
+Set the following Grid environment variables to simplify entering the
+`grid` commands in this procedure.
+
+{:start="3"}
+
+3.  Set `GRID_DAEMON_KEY` to the base name of the agent's public/private key
+    files (such as `alpha-agent`). This environment variable replaces the
+    `-k` option on the `grid` command line.
+
+    Tip: If you're using this example docker-compose environment, this
+    variable is already defined for the `gridd-alpha` and `gridd-beta` containers.
+
+    ```
+    root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+    ```
+
+    **Note**: Although this variable has "daemon" in the name, it should
+    reference the user key files in `$HOME/.grid/keys`, not the Grid daemon's key
+    files in `/etc/grid/keys`.
+
+1.  Set `GRID_DAEMON_ENDPOINT` to the endpoint for the node's `gridd` container
+    (such as `http://localhost:8080`). This environment variable replaces the
+    `--url` option on the `grid` command line.
+
+    Tip: If you're using this example docker-compose environment, this variable
+    is already defined for the `gridd-alpha` and `gridd-beta` containers.
+
+    ```
+    root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+    ```
+
+1.  For Grid on Splinter: Set `GRID_SERVICE_ID` to the fully qualified service ID
+    for the scabbard service on this circuit (such as `01234-ABCDE::gsAA`).
+    This environment variable replaces the `--service_id` option on the `grid`
+    command line.
+
+    ```
+    root@gridd-alpha:/# export GRID_SERVICE_ID=01234-ABCDE::gsAA
+    ```
+
+1.  Follow this same procedure in a separate command line for the `gridd-beta`
+    container.
+
+    ```
+    $ docker exec -it gridd-beta bash
+    ```
+
+### Create a "Buyer" Organization
+
+{:start="6"}
+
+6.  Using the `gridd-alpha` command line, create a new organization by
+    specifying a unique organization ID, and the organization's name.
+
+    ```
+    root@gridd-alpha:/# grid organization create \
+    farm "Fred's Farm"
+    ```
+
+   This command creates and submits a transaction to create a new Pike
+   organization with the data you supplied, as well as a new Pike agent
+   with the `admin` role and `active` status.
+
+   The transaction is signed with the agent's private key, as derived from the
+   base name specified by `GRID_DAEMON_KEY` (or with the `-k` option on the
+   command line). The agent's public key is used as the agent ID.
+
+   **Note**: This command doesn't display any output. Instead, check the log
+   messages (in the terminal window where you started the Grid Docker
+   environment) for the success or failure of this operation.
+
+#### Create a Purchase Order Admin Role
+
+In order to submit transactions using other Grid features the submitting
+agent
+must possess an active role with the appropriate permissions.
+
+{:start="7"}
+
+7. Verify that the agent's public key file exists in
```

Comment:
Should this maybe be moved down between the two bottom steps (since it seems like the contents of that pub key are used in the next next command, not the next command like this makes it sound.

Reply:
This file is necessary for all grid commands. In the docker environment, this is handled with the `GRID_DAEMON_KEY` env variable to replace the `-k` flag in the commands. The `agent` commands require that a public key be specified though so that's what that `cat` section is for